astro/src/content/docs/apis/_users-response-body.mdx

```diff
@@ -24,218 +24,20 @@ import RemovedSince from 'src/components/api/RemovedSince.astro';
   </APIField>
 
   <APIField name="total" type="Long" renderif={!!props.search_result}>
-    The number of users that matched the search criteria. The number of results in the response will be equal to the <code>numberOfResults</code> provided on the request, if not provided the default number of results will be returned.
+    The number of users that matched the search criteria. The number of results in the response will be equal to the `numberOfResults` provided on the request, if not provided the default number of results will be returned.
```

Comment:
Shouldn't this use `InlineField` because they refer to another field name?
```suggestion
    The number of users that matched the search criteria. The number of results in the response will be equal to the <InlineField>numberOfResults</InlineField> provided on the request, if not provided the default number of results will be returned.
```

Reply:
I didn't edit this field originally but the linter made me fix it since the whole file has to be fixed if any part is touched. Fixed